### ISSUE TITLE
Cyclet usage without React.createClass?

### ISSUE THREAD

**Issue by girvo** [RGBz/Unicycle#2]
Hey there, lovely library.

I'm curious whether there's a supported way of using Cyclet with ECMAScript 6 class-based React components? They don't really support mixins, is there another way of wiring the Stores and Components together?

Cheers

**Comment by RGBz** [comment]
Hello and thanks!

I'm about to head to bed, but I'll see what I can do in the AM. I'm new to ES6, but I'm sure I can figure it out. I'd want to do something that's backwards compatible with the existing API. 

I'll look into it first thing tomorrow.

**Comment by RGBz** [comment]
@girvo,

I just updated the API to version 1.2 and added a new `Cyclet.connectWithStores()` method. The [README](/RGBz/cyclet/#readme) has been updated with a new example and the API reference for how to use it. Despite the update, Cyclet is still backwards compatible with the Mixin approach from 1.1. Basically the method wraps a React Component with a new one that listens on the stores you want and passes updated store state to your component as props. It's a significantly different approach, but I think it's cleaner.

Thanks for the request! Let me know if you run into any issues.
